index.js

```diff
@@ -632,6 +632,54 @@
 
   //. ### Function
 
+  //# curry2 :: ((a, b) -> c) -> a -> b -> c
+  //.
+  //. Curry a binary function.
+  //. ```javascript
```

Comment:
Please include an "empty" (`//.`) line between the description and the code block.